guides/common/modules/con_http-booting.adoc

```diff
@@ -3,4 +3,16 @@
 [id="http-booting"]
 = HTTP booting
 
-You can use HTTP booting to boot systems over a network using HTTP.
+The term HTTP booting refers to two booting stages _UEFI HTTP Boot_ and _Boot Loader HTTP Boot_.
```

Comment:
```suggestion
The term HTTP booting refers to two booting stages: UEFI HTTP Boot and Boot Loader HTTP Boot.
```

In general, we only use italics for items that users have to replace. -> I suggest to drop the emphasis.